pkg/virtctl/vmexport/vmexport.go

```diff
@@ -606,6 +772,11 @@ func getExportSource() k8sv1.TypedLocalObjectReference {
 	return exportSource
 }
 
+func isVmExportSource(source k8sv1.TypedLocalObjectReference) bool {
```

Comment:
You can export a PVC that is NOT part of a VM, so PVCs is technically not a VM, but I guess if you read vmExport Source then it makes no sense. Let me see if I can name it better

Reply:
With the switch to a flag instead of an argument. We now have a check in the download argumentation validation that will not allow `--manifest` and `--pvc` to co-exist with the appropriate message.